_posts/2022-09-13-launching-community-built-implementation-guide.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Launching a community built implementation guide for the Standard for Public Code"
+date: 2022-09-13
+author: Jan Ainali
+type: blogpost
+excerpt: Helping you on your way to fully adapting the Standard for Public Code for your codebase.
+categories:
+- News
+---
+
+# Launching a community built implementation guide for the Standard for Public Code
+
+## Background
+
+Almost since the beginning of the Standard for Public Code, we have had discussions about how we can make it easier for people to start applying it to their codebase.
```

Comment:
```suggestion
Almost since the beginning of the [Standard for Public Code](https://standard.publiccode.net/), we have had discussions about how we can make it easier for people to start applying it to their codebase.
```